src/Test/IntegrationTestCasePhpUnit10.php

```diff
@@ -0,0 +1,51 @@
+<?php
+
+/*
+ * This file is part of Twig.
+ *
+ * (c) Fabien Potencier
+ *
+ * For the full copyright and license information, please view the LICENSE
+ * file that was distributed with this source code.
+ */
+
+namespace Twig\Test;
+
+/**
+ * Integration test helper.
+ *
+ * @author Fabien Potencier <fabien@symfony.com>
+ * @author Karma Dordrak <drak@zikula.org>
+ */
+abstract class IntegrationTestCasePhpUnit10 extends BaseIntegrationTestCase
```

Comment:
I really don't like this class' name.

Reply:
I don't have a good idea for the name, but this one is indeed not good. This class works on *all* PHPUnit versions, not just on PHPUnit 10 (static data providers have always been supported by PHPUnit since data providers exist). And it is the class that we will want projects to use (the other one should be marked as deprecated at some point)